hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -155,14 +156,16 @@ private void initNameNode(ExecutorDriver driver, String dnsName) throws Exceptio
         // is used as a mutex indicates whether or not any NameNode has ever been formatted.
         // The first NameNode to acquire the mutex and find that no NameNode has ever been
         // formatted, formats itself.  All others bootstrap.
```

Comment:
Hdfs-site.xml looks like following:
If backup is disabled - dfs.namenode.name.dir = $dataDir/name
If backup is enabled - dfs.namenode.name.dir = $dataDir/name,/nfs/hadoop/namenode1 for NN1
and $dataDir/name,/nfs/hadoop/namenode2 for NN2.
Note, that $backupDir variable in hdfs-site.xml already contains "namenode1|2" suffix. 

There and why do you think we need those check?